lib/versions.js

```diff
@@ -106,14 +106,13 @@ function fetchLatestTag(repo) {
   );
 }
 
-async function getVersionFromToolVersionsFile(toolVersionsPath) {
+async function getVersionFromToolVersionsFile(toolVersionsPath = ".tool-versions") {
```

Comment:
same story as in #28: this code doesn't cover this case:

```js
"" || "foo" == "foo"
```

which we actually need, as it is pretty easy to declare the input but not set it in yaml